src/backend/distributed/sql/downgrades/citus--11.0-4--11.0-3.sql

```diff
@@ -0,0 +1 @@
+#include "../udfs/citus_finish_pg_upgrade/11.0-3.sql"
```

Comment:
```
/home/circleci/project/src/backend/distributed/sql/downgrades/citus--11.0-4--11.0-3.sql:1:10: fatal error: ../udfs/citus_finish_pg_upgrade/11.0-3.sql: No such file or directory
    1 | #include "../udfs/citus_finish_pg_upgrade/11.0-3.sql"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This needs an update here.

```suggestion
#include "../udfs/citus_finish_pg_upgrade/11.0-1.sql"
```